contracts/StablecoinConverter.sol

```diff
@@ -347,6 +326,18 @@ contract StablecoinConverter is EpochTokenLocker {
         }
     }
 
+    function updateTokenConservation(
+        int[] memory tokenConservation,
+        Order memory order,
+        uint16[] memory tokenIdsForPrice,
+        uint128 buyAmount,
+        uint128 sellAmount
+    ) private pure returns(int[] memory) {
+        tokenConservation[findPriceIndex(order.buyToken, tokenIdsForPrice)] -= int(buyAmount);
```

Comment:
I didn't know about `int`... what does it do?

Reply:
https://solidity.readthedocs.io/en/v0.5.3/types.html#integers